.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x, 16.x]
```

Comment:
that's not equivalent to the current testing strategy
```
  - "node"  # Latest
  - "lts/*" # Latest LTS
  - "12"
  - "11"
  - "10"
  - "9"
  - "8"
```

Reply:
there is no support for LTS in github actions it seems, tried it manually and searched for issues
https://github.com/actions/setup-node/issues/26
(i think this is better, prevents a new PR just randomly failing in the future)